README.md

```diff
@@ -93,6 +93,7 @@ You can either specify specific Julia versions or version ranges. If you specify
 - `^1.3.0-0` is a **caret** version range that includes _all_ pre-releases. It matches all versions `≥ 1.3.0-` and `< 2.0.0`.
 - `~1.3.0-0` is a **tilde** version range that includes _all_ pre-releases. It matches all versions `≥ 1.3.0-` and `< 1.4.0`.
 - `nightly` will install the latest nightly build.
+- `1.6-nightly` will install the latest nightly build for the upcoming 1.6 release. This version will only be available during certain phases of the Julia release cycle.
```

Comment:
@maleadt Do you know if these nightly builds are documented anywhere? https://julialang.org/downloads/nightlies/ doesn't mention them and it would be nice to link to a page where people can check what nightly builds are available.

Reply:
I don't think these are documented anywhere, except in code: https://github.com/JuliaCI/julia-buildbot/blob/37ef3e30aeeccd172f4b9c27749e70dc9f701d54/master/master.cfg#L120-L128 (and the `majmin` handling across those files).

Reply:
@staticfloat

Reply:
Yeah, they're not really documented anywhere, but you can download a `latest` build of Julia overall, as well as from one of the `$(major).$(minor)` directories.